cmd/podman/machine/set.go

```diff
@@ -86,6 +87,9 @@ func init() {
 	userModeNetFlagName := "user-mode-networking"
 	flags.BoolVar(&setFlags.UserModeNetworking, userModeNetFlagName, false, // defaults not-relevant due to use of Changed()
 		"Whether this machine should use user-mode networking, routing traffic through a host user-space process")
+
+	rosettaFlagName := "rosetta"
+	flags.BoolVar(&setFlags.Rosetta, rosettaFlagName, true, "Whether this machine should use Rosetta, running x86_64 Linux binaries under ARM Linux on Apple silicon")
```

Comment:
again, i think this probably should go to containers.conf

Reply:
fixed. removed `--rosetta` option.